scripts/utilities/package_dataset_for_runpod.py

```
@@ -0,0 +1,754 @@
+    """Find the actual image file, handling encoding mismatches"""
+    # Try the original path first
+    source_path = Path(image_path)
+    if source_path.exists():
+        return source_path
+
+    # Try with normalized path (reverse of what we did for captions)
+    # The caption may have been normalized FROM the file path, so reverse it
+    reversed_replacements = {
+        "ch�teau": "château",
+        "�": "è",
+        "�": "é",
```

Ruff (0.12.2)

Dictionary key literal `"�"` repeated

(F601)